Runtime_Support/dot_net_core.md

```diff
@@ -1,25 +1,34 @@
-# .NET Core on App Service
+# .NET on App Service
 
-## .NET Core Update Policy
+## .NET Update Policy
 
-App Service upgrades the underlying .NET Core runtime and SDK of your application as part of the regular platform updates. As a result of this update process, your application will be automatically updated to the latest patch version available in the platform for the configured minor version of your app.
+App Service upgrades the underlying .NET runtime and SDK of your application as part of the regular platform updates. As a result of this update process, your application will be automatically updated to the latest patch version available in the platform for the configured minor version of your app.
```

Comment:
"as a result of this update" - you already said "process"

Reply:
fixed